leak_suppr.txt

```diff
@@ -0,0 +1 @@
+leak:style::sharing::SHARING_CACHE_KEY::__init
```

Comment:
Please move this file to the `support` directory and call it "suppressed_leaks_for_asan.txt".

Reply:
Just noticed that the file was removed during refactoring instead of renamed. Follow-up PR to fix that: #32527